src/global_logic.js

```diff
@@ -1850,6 +1860,21 @@ module.exports.addSkilldataToTotals = function (totals, comb, arml, buff) {
                                     buff["enemyDebuffCount"] = arm[skillkey + "Detail"];
                             }
                         }
+                    } else if (stype === 'epic' && key === 'Djeeta') {
+                        // This implementation assume 'Djeeta'
+                        // To collect skill information only once per grid.
+                        // skill are affect to all allies
+                        if (amount === 'count-epic') {
+                            numGrandEpic += comb[i];
+                            countEpic = countEpic ? countEpic : epic.countEpicWeapon(arml, comb);
+                        } else if (amount === 'count-wand') {
+                            numResonanceStaff += comb[i];
+                            countWand = countWand ? countWand : epic.countWandType(arml, comb);
+                        } else if (amount === 'all-unique-type') {
+                            isAllUniqueArmType = epic.isAllUniqueArmType(arml, comb);
+                        } else if (amount === 'all-unique') {
+                            isAllUniqueArm = epic.isAllUniqueArm(arml, comb);
+                        }
```

Comment:
Checkpoints (confirm)
those local variables are all override
- [ ] `numGrandEpic` and `numResonanceStaff` are stack. (count staff here but the effect will not stack)
- [ ] `isAllUniqueArmType`, `isAllUniqueArm` override but logically they will be `false` if two or more the same skill.

Reply:
those `=` (override) code it's about run-time cost optimization.
no need to re-collect weapon's information. it's enough only the first time. 

- [x] `countEpic' `countWand' avoid the 2nd time check ... ok
- [ ] `isAllUniqueArmType` `isAllUniqueArm` re-evaluate then set false